### ISSUE TITLE
Migrate temp dir

### ISSUE THREAD

**Issue by emersion** [arecker/bennedetto#5]
Fresh install. Running `mkdir temp` fixed the issue.

```
Cleaning up...
python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/simon/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/home/simon/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 328, in execute
    django.setup()
  File "/home/simon/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/__init__.py", line 17, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/simon/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/utils/log.py", line 86, in configure_logging
    logging_config_func(logging_settings)
  File "/usr/lib/python2.7/logging/config.py", line 794, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/lib/python2.7/logging/config.py", line 576, in configure
    '%r: %s' % (name, e))
ValueError: Unable to configure handler 'file': [Errno 2] No such file or directory: '/home/simon/software/bennedetto/temp/debug.log'
Makefile:9: recipe for target 'migrate' failed
make: *** [migrate] Error 1
```

**Comment by arecker** [comment]
Thanks for reporting this.  Probably just needs either a `.gitignore` or a `.gitkeep` in that directory

**Comment by emersion** [comment]
Maybe it will be safer to create it automatically, since we sometimes need to clear temp files and run `rm -rf temp`?

**Comment by danrue** [comment]
See #6 for my suggestion.

**Comment by arecker** [comment]
Fixed by #6
